src/screens/ConstructionSiteDetailScreen.js

```diff
@@ -0,0 +1,162 @@
+  const endTime = endDate ? moment(endDate, 'YYYY-MM-DD HH:mm:ss').format('HH:mm') : undefined;
+
+  if ((endTime !== undefined && endTime !== '00:00') || startTime !== '00:00') {
+    return formattedEndDate && endTime
+      ? `${formattedStartDate} ${startTime} - ${formattedEndDate} ${endTime}`
+      : `${formattedStartDate} ${startTime}`;
+  }
+
+  return formattedEndDate ? `${formattedStartDate} - ${formattedEndDate}` : formattedStartDate;
+};
+
+export const ConstructionSiteDetailScreen = ({ navigation }) => {
+  const { constructionSites } = useContext(ConstructionSiteContext);
+
+  const index = navigation.getParam('index');
+
+  useMatomoTrackScreenView(MATOMO_TRACKING.SCREEN_VIEW.CONSTRUCTION_SITE_DETAIL);
+
+  if (typeof index !== 'number') return null;
+
+  const {
+    category,
+    cause,
+    description,
+    direction,
+    endDate,
+    imageUri,
+    location,
+    locationDescription,
+    restrictions,
+    startDate,
+    title
+  } = constructionSites[index];
+
+  const extendedTitle = (category ? `${category} - ` : '') + title;
+  const formattedDates = formatDates(startDate, endDate);
+
+  return (
+    <SafeAreaViewFlex>
+      <ScrollView keyboardShouldPersistTaps="handled">
+        <WrapperWithOrientation>
+          {!!imageUri && <Image source={{ uri: imageUri }} />}
+          <TitleContainer>
+            <Title>{extendedTitle}</Title>
+          </TitleContainer>
+          <Wrapper>
+            <View style={styles.informationContainer}>
+              <BoldText>{endDate ? 'Gesamtzeitraum: ' : 'Datum: '}</BoldText>
+              <RegularText>{formattedDates}</RegularText>
+            </View>
+            {!!locationDescription && (
+              <View style={styles.informationContainer}>
+                <BoldText>Standort: </BoldText>
+                <RegularText>{locationDescription}</RegularText>
+              </View>
+            )}
+            {!!direction && (
+              <View style={styles.informationContainer}>
+                <BoldText>Richtung: </BoldText>
+                <RegularText>{direction}</RegularText>
+              </View>
+            )}
+            {!!cause && (
+              <View style={styles.informationContainer}>
+                <BoldText>Ursache: </BoldText>
+                <RegularText>{cause}</RegularText>
+              </View>
+            )}
+            {!!description && (
+              <View style={styles.verticalPadding}>
+                <BoldText>Weitere Informationen: </BoldText>
+                <RegularText>{description}</RegularText>
+              </View>
+            )}
+            {!!restrictions?.length && (
+              <View style={styles.verticalPadding}>
+                <BoldText>Aktuelle Einschränkungen: </BoldText>
+                {restrictions.map((restriction) => (
+                  <RegularText key={'restriction-' + index}>- {restriction}</RegularText>
```

Comment:
use template-string here like in other places `restriction-${index}`? is eslint complaining here?

Reply:
a) i do not follow the first question. can you please rephrase?
b) eslint does not complain, should it?

Reply:
changed it in 576046e1ec323823d7efca37960b378005f213d7

Reply:
i thought i know messages from eslint about that. but maybe from another project.
so we are cool with having both options over the code base here, right?

Reply:
:+1: